views/indomie-chicken-veggie-stir-fry.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Indomie Chicken & Veggie Stir Fry</title>
+  </head>
+  <body>
+    <h1>Noodles!</h1>
+    <h2>Indomie Chicken & Veggie Stir Fry</h2>
+    <img
+      src="../assets/indomie-chicken-veggie-stir-fry_detail2_215552305.png"
+      style="width: 362px; height: 312px"
```

Comment:
great use of inline style :) however, for img tag, you can consider using the native `width` and `height` attribute instead 

https://www.w3schools.com/tags/att_img_width.asp